### ISSUE TITLE
tamu-library-components should have specified accessibility design standards

### ISSUE THREAD

**Issue by jeremythuff** [TAMULib/tamu-library-components#21]
blocks tamulib/weaver-components#8

[display/Applications/Accessibility](https://wikis.library.tamu.edu/display/Applications/Accessibility)
